antsibull/cli/doc_commands/sphinx_init.py

```diff
@@ -0,0 +1,96 @@
+# Author: Felix Fontein <felix@fontein.de>
+# License: GPLv3+
+# Copyright: Ansible Project, 2021
+"""Entrypoint to the antsibull-docs script."""
+
+import os
+import os.path
+
+from ... import app_context
+from ...jinja2.environment import doc_environment
+from ...logging import log
+
+
+mlog = log.fields(mod=__name__)
+
+
+TEMPLATES = [
+    '.gitignore',
+    'build.sh',
+    'conf.py',
+    'requirements.txt',
+    'rst/index.rst',
+]
+
+
+def write_file(filename: str, content: str) -> None:
```

Comment:
Have you considered invoking cookiecutter instead of implementing own templating?

Reply:
> I don't like including a dependency for something that simple.

That's understandable but then you'd also gain some of the more advanced features like testing your skeleton generator.